.gitignore

```diff
@@ -3,7 +3,7 @@ syntax: glob
 ### VisualStudio ###
 
 # Tool Runtime Dir
-/[Tt]ools/
+[Tt]ools/
```

Comment:
You are correct, for now I am reverting this change and adding comments. In later discussions we might consider renaming our tools directories.